test/spec/modules/userId_spec.js

```diff
@@ -2754,6 +2732,31 @@ describe('User ID', function () {
             done();
           });
         });
+        it('pbjs.getUserIdsAsEidBySource', function () {
+          var users = [
+            {
+              'source': 'pubcid.org',
+              'uids': [
+                {
+                  'id': '11111',
+                  'atype': 1
+                }
+              ]
+            }
+          ];
+          setSubmoduleRegistry([sharedIdSystemSubmodule]);
+          init(config);
+          config.setConfig({
+            userSync: {
+              syncDelay: 0,
+              userIds: [{
+                name: 'pubCommonId', value: {'pubcid': '11111'}
+              }]
```

Comment:
should you add another id module here so that your test confirms it only picks the source you specify? right now, if it returned the entire eids array it would still pass.

Reply:
Added one more module. Test results are passed by adding one more module